CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.46.0 / 2021-02-24
+
```

Comment:
Can we add information that we are looking for prometheus-operator adopters entries in ADOPTERS.md file?

Reply:
sure thing, you mean as a changelog entry? I am not quite clear where and how I should put that request.

Reply:
not sure I would add it to the changelog. Maybe mention it when we announce the release?

Reply:
It doesn't need to be in the changelog as this is something more for a release note.

Reply:
sure thing, I can add it in the release note section in github, where we usually copy the contents of the changelog. is that a better place?